etc/taskcluster/decision_task.py

```diff
@@ -103,6 +103,30 @@ def android_x86():
         )
         .find_or_create("build.android_x86_release." + CONFIG.git_sha)
     )
+    return (
```

Comment:
(ignore if you disagree) can we split this into an android_x86_build and android_x86_test function?

Reply:
I expect we’ll never want to build for Android x86 other than in order to run tests in the emulator (devices we care about are all ARM or ARM64), but if we ever do it’ll be easy to split then.